setup.py

```diff
@@ -4,6 +4,9 @@
 with open("README.md", "r", encoding="utf-8") as fh:
     long_description = fh.read()
 
+with open('requirements.txt') as f:
+    required = f.read().splitlines()
```

Comment:
in all other packages we add a `strip` to deal with rogue spaces, WDYT ?

```suggestion
with open('requirements.txt') as f:
    required = [c.strip() for c in f.readlines()]
```

Reply:
Makes sense!